### DISCUSSION TITLE
新玩具-Keychron-K3-Pro | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#166] [Announcements]
https://blog.goalonez.site/blog/%E6%96%B0%E7%8E%A9%E5%85%B7-Keychron-K3-Pro.html 

 Goalonez Blog

**Comment by luxuxl** [comment]
博主, 我没用过这种 Via 改键的键盘, 请问可以实现哪些功能, 我一般用的是软件改键, 如 Mac 上的 Karabiner

我喜欢把 Caps + wsad 改为方向键, Via 改键能实现吗
